docs-search/common/src/Docs/Search/Declarations.purs

```diff
@@ -201,19 +211,15 @@ getLevelAndName (Declaration { info, title }) =
 -- | Extract package name from `sourceSpan.name`, which contains path to
 -- | the source file. If `ModuleName` string starts with `Prim.`, it's a
 -- | built-in (guaranteed by the compiler).
-extractPackageName :: ModuleName -> Maybe SourceSpan -> PackageInfo
-extractPackageName (ModuleName moduleName) _
-  | String.split (Pattern ".") moduleName !! 0 == Just "Prim" = Builtin
-extractPackageName _ Nothing = UnknownPackage
-extractPackageName _ (Just (SourceSpan { name })) =
-  fromMaybe LocalPackage do
-    topLevelDir <- dirs !! 0
-    if topLevelDir == ".spago" then Package <<< PackageName <$> dirs !! 2
-    else do
-      bowerDirIx <- Array.findIndex (_ == "bower_components") dirs
-      Package <<< PackageName <$> dirs !! (bowerDirIx + 1)
-  where
-  dirs = String.split (Pattern "/") name
+extractPackageName :: Graph.ModuleGraphWithPackage -> Set PackageName -> ModuleName -> PackageInfo
+extractPackageName moduleGraph workspacePackages (ModuleName moduleName) =
+  case String.split (Pattern ".") moduleName !! 0 == Just "Prim" of
```

Comment:
```suggestion
  case moduleName # startsWith "Prim." of
```

Reply:
I committed this but I'm reverting it - `Prim` is a builtin module and it's not matched by the `startsWith` implementation